styles.css

```diff
@@ -256,35 +232,23 @@ h2 {
   height: auto;
 }
 
-.card {
+.cards-section {
   /* width: 100%; */
   background-color: white;
   padding: 2%;
   border-radius: 15px;
-  margin: 5% 5% 5% 5%;
-  padding-left: 7%;
 }
 
-.projects-done h2 {
+.one h2 {
   display: flex;
   justify-content: flex-start;
-  font-family: Arial, Helvetica, sans-serif;
   font-style: normal;
   font-weight: 700;
-  font-size: 32px;
+  font-size: 24px;
   line-height: 44px;
 }
 
-.projects-done .time {
-  display: flex;
-  flex-direction: row;
-  align-items: center;
-  gap: 12px;
-  width: 236px;
-  height: 32px;
-}
```

Comment:
- [ ] Please make sure you sticked to the figma design and aligned the items to the center and create a form as the template. if you follow the figma design its good for your designing and future skills. give it `background-repeat: no-repeat; background-size: cover;`
## your design👇
![contactst](https://user-images.githubusercontent.com/66992901/194420635-e46cd14c-4364-4176-b7d4-7cf516fecd58.PNG)
## figma design👇
![lastfigma](https://user-images.githubusercontent.com/66992901/194420813-250e7633-f1cf-472c-9d9d-2a1fd0bb8043.PNG)